src/xdocs/google_style.xml

```diff
@@ -2437,6 +2437,31 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
+          <br/>
+          <br/>
+          <code>org.checkstyle.google.severity=${SeverityLevel}</code>
+          <br/>
+          <br/>
+          where <code>SeverityLevel</code> can be any one of them:
+          <code>ignore</code>, <code>info</code>, <code>warning</code>, <code>error</code>.
+          Default is <code>Warning</code>.
+        </p>
+        <p>
+          Visit here for more details about Severity Levels:
+          <a href="https://checkstyle.sourceforge.io/property_types.html#SeverityLevel">
+            Severity Level
+          </a>
+        </p>
+        <p>
+          Visit here for more details about how to use custom properties:
+          <a href="https://checkstyle.org/cmdline.html#Command_line_usage">
+            Command Line Usage
+          </a>
```

Comment:
please add link to https://checkstyle.org/anttask.html#Parameters `properties` parameter.
To let user know how to set it.

Reply:
Or just:
`Custom properties can be set by property file in CLI, properties parameter in Ant task, or as system variable`
with links to specific pages on words in this sentense.

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/c73f45d0baa6ed8d19f4edcc0fdf13840bcb9654/src/xdocs/google_style.xml#L2453-L2462

Reply:
Website: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/c73f45d_2024154035/google_style.html#Google_Overrides